### ISSUE TITLE
tridentctl install --image-registry does not update image for netapp-autosupport

### ISSUE THREAD

**Issue by JanusTekk** [NetApp/trident#715]
**Describe the bug**
Using the option to supply a private registry when installing v22.01.1 results in backoffs for ErrImagePull since the reference for netapp-autosupport does not update correctly.

**Environment**
I'm in an air gapped environment, the registry flag is required to point to our own registry where I have pre-populated the images

- Tridentctl version: v22.01.1
- Trident installation flags used: -n trident-ns --image-registry private-registry.organisation/repo

**To Reproduce**
- Download and extract the release for v22.01.1
- run `tridentctl install -n trident-ns --image-registry private-registry.organisation/repo` 
- pods fail to start with errors that netapp/netapp-autosupport cannot be pulled

**Expected behavior**
The pods trident tries to deploy reference private-registry.organisation/repo/netapp/<image:tag>

**Additional context**

**Comment by lukasz-borek** [comment]
+1, my fix :

```
VERSION=22.01.1
for IMAGE_BASENAME in trident trident-operator trident-autosupport
do
if [ "$IMAGE_BASENAME" = "trident-autosupport" ]; then
  VERSION="22.01"
fi
...
```

**Comment by gnarl** [comment]
This was fixed with [commit 7eaca8f](https://github.com/NetApp/trident/commit/7eaca8fd120de60a4a96a1c4a0f8e94705778b65) and is included in the Trident v22.04.0 release.
